spring-core/src/main/java/org/springframework/cglib/core/DuplicatesPredicate.java

```diff
@@ -89,14 +89,12 @@ public DuplicatesPredicate(List allMethods) {
             continue;
           }
           InputStream is = cl.getResourceAsStream(c.getName().replace('.', '/') + ".class");
-          if (is == null) {
```

Comment:
Please don't change files in the cglib package, this part is manually shaded into our repository and we shouldn't change the source if we want to easily keep up with the original.